docs/source/developers/continuous_integration/faqs.rst

```diff
@@ -0,0 +1,45 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+.. or more contributor license agreements.  See the NOTICE file
+.. distributed with this work for additional information
+.. regarding copyright ownership.  The ASF licenses this file
+.. to you under the Apache License, Version 2.0 (the
+.. "License"); you may not use this file except in compliance
+.. with the License.  You may obtain a copy of the License at
+
+..   http://www.apache.org/licenses/LICENSE-2.0
+
+.. Unless required by applicable law or agreed to in writing,
+.. software distributed under the License is distributed on an
+.. "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+.. KIND, either express or implied.  See the License for the
+.. specific language governing permissions and limitations
+.. under the License.
+
+
+FAQs
+====
+
+Q: What scripts are run when I make changes to an open PR?
+
+
+Q: What scripts are run when a PR is merged?
+
+(e.g. the dev PR)
+
+
+
+Q: What is the cpp_cron workflow, when does it run, and why does it exist?
+https://github.com/apache/arrow/blob/master/.github/workflows/cpp_cron.yml
+
+High-level overview:
+
+    Possibly add diagram: https://lucid.app/lucidchart/ea221ae9-afc5-4453-8939-970e08759127/edit?shared=true&page=0_0#
+    Short description of each component
+
+
+Generally add more content to make it easier to get started with understanding CI.
+
+
+
+Potentially add something about how linx-apt-r (etc) depend on other dockerfiles
+
```

Comment:
For now I came up with:
- Where to find more info about the Archery/Crossbow tools? (provide a link).
- Different CI services - don't know if there is a need of explaining, but I did question why it is so.